src/Symfony/Component/Yaml/Parser.php

```diff
@@ -1225,6 +1225,10 @@ private function lexUnquotedString(int &$cursor): string
         $offset = $cursor;
         $cursor += strcspn($this->currentLine, '[]{},: ', $cursor);
 
+        if ($cursor === $offset) {
+            throw new ParseException('Malformed unquoted YAML string');
```

Comment:
I know that most other exceptions in this file does not end with a period. But all new exceptions should. 

```suggestion
            throw new ParseException('Malformed unquoted YAML string.');
```

Reply:
Alright! Wasn't sure what to do with the period fabbot proposed when I first created the PR. Will know for the future, fixed!